README.md

```diff
@@ -1,15 +1,32 @@
 # Multiuser Whiteboard
 
+[![Github Actions Badge](https://github.com/InvictoProjects/Multiuser_Whiteboard/workflows/Java%20CI%20with%20Maven/badge.svg)](https://github.com/InvictoProjects/Multiuser_Whiteboard/actions?query=workflow%3A%22Java+CI+with+Maven%22)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fed4c742f21e4c69b22a6b55c9ff0710)](https://app.codacy.com/gh/InvictoProjects/Multiuser_Whiteboard?utm_source=github.com&utm_medium=referral&utm_content=InvictoProjects/Multiuser_Whiteboard&utm_campaign=Badge_Grade_Settings)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
 ## Relevance
 Online learning is becoming more popular today. No doubt, it is both a convenience and a challenge. It gives you a great deal of flexibility, but with that freedom sometimes you need to solve more different problems such as organizing of online learning and convenient presentation of information. For this reason, our team is developing an application that will make your online learning better.
 
-## Concept 
-Multiuser Whiteboard is a web application that helps you communicate more easily with other people by visualizing your thoughts on an interactive whiteboard. Users are able to create or join virtual rooms, they can watch and interact with a shared graphics editor. Each user also has the opportunity to communicate through voice and text chats.
+## Concept
+Multiuser Whiteboard is a web application that helps you communicate more easily with other people by visualizing your thoughts on an interactive whiteboard. Users are able to create or join virtual rooms, they can watch and interact with a shared graphics editor. Each user also has the opportunity to communicate through text chat.
+
```

Comment:
```suggestion
## About Program
MULTIUSER WHITEBOARD is a free and open online board for many users to work together. Here you can draw your ideas and discuss them with others. The board is updated in real time for all connected users. It can be used for various purposes such as art, entertainment, design and education. To work together on the board, just share the URL to others. You can create a personal board with a random name, which will only be available at the appropriate link, and you can join a specific board by its name.

The program implements its own server based on the ServerSocket Java class. When a user connects to the board, a WebSocket is initialized, which allows all participants on a particular board to receive changes in real time. The program uses asynchronous programming and multithreading.
```